src/Sarif.Converters/AndroidStudioConverter.cs

```diff
@@ -151,14 +151,15 @@ public Result ConvertProblemToSarifResult(AndroidStudioProblem problem)
             {
                 location.PhysicalLocation = new PhysicalLocation
                 {
+                    FileLocation = new FileLocation(),
```

Comment:
This will lead to the same problem we had in the other review (`"fileLocation": {}`) if `uri` is `null`. Is `uri` guaranteed non-`null` here? If not, only populate `FileLocation` if `uri` is non-`null`.

Reply:
Yes it will be non-null